gigareview/console/tests/acceptance/FetchReports.feature

```diff
@@ -5,6 +5,11 @@ Feature: FetchReports
 
   Scenario: try FetchReports
     Given a EM report is uploaded daily to a sftp server
```

Comment:
If you look at corresponding `aEMReportIsUploadedDailyToASftpServer()` in `AcceptanceTester`, the function checks if all of the previous day's EM reports are present on the production SFTP server, not just one single EM report. This `Given` statement could be more descriptive as to what the function is doing.